URBNSwiftAlert/Classes/URBNSwAlertStyler.swift

```diff
@@ -0,0 +1,421 @@
+    
+    /**
+     * Font of the alert's title
+     */
+    public var titleFont: UIFont? = UIFont.systemFont(ofSize: 14)
+    
+    /**
+     * Font of the alert's message
+     */
+    public var messageFont: UIFont? = UIFont.systemFont(ofSize: 12.0)
+    
+    /**
+     * Text color of the alert's title
+     */
+    public var titleColor = UIColor.black
+    
+    /**
+     * Text color of the alert's message
+     */
+    public var messageColor = UIColor.black
+    
+    /**
+     * Background color of the alert's title
+     */
+    public var titleBackgroundColor = UIColor.clear
+    
+    /**
+     * Background color of the alert's message
+     */
+    public var messageBackgroundColor = UIColor.clear
+    
+    /**
+     * Alignment of the titles's message
+     */
+    public var titleAlignment = NSTextAlignment.center
+    
+    /**
+     * Alignment of the alert's message
+     */
+    public var messageAlignment = NSTextAlignment.center
+    
+    /**
+     * Max input length for the text field when enabled
+     */
+    public var textFieldMaxLength = Int.max
+    
+    /**
+     * Text color of the error label text
+     */
+    public var textFieldErrorMessageColor = UIColor.red
+    
+    /**
+     * Text font of the error label text
+     */
+    public var textFieldErrorMessageFont: UIFont? = UIFont.systemFont(ofSize: 12.0)
+    
+    /**
+     *  Text Insets for input text fields on alerts
+     */
+    public var textFieldEdgeInsets = UIEdgeInsets(top: 0, left: 15, bottom: 0, right: 15)
+
+    /**
+     * Corner radius of the alert view itself
+     */
+    public var alertCornerRadius: CGFloat = 0.0
+    
+    /**
+     *  Vertical margin between textfields
+     */
+    public var textFieldVerticalMargin: CGFloat = 5.0
+    
+    /**
+     *  Layout axis for buttons (for 3+ always vertical always used)
+     */
+    public var buttonsLayoutAxis = UILayoutConstraintAxis.horizontal
+    
+    /**
+     *  Layout Margins for the alert title
+     */
+    public var titleLabelInsetConstraints = InsetConstraints(insets: UIEdgeInsets.zero, priority: UILayoutPriorityDefaultHigh)
+    
+    /**
+     *  Layout Margins for the alert message
+     */
+    public var messageInsetConstraints = InsetConstraints(insets: UIEdgeInsets.zero, priority: UILayoutPriorityDefaultHigh)
+    
+    /**
+     * UIEdgeInsets used at the external margins for the buttons of the alert's buttons
+     */
+    public var buttonInsetConstraints = InsetConstraints(insets: UIEdgeInsets.zero, priority: UILayoutPriorityDefaultHigh)
+    
+    /**
+     * Width of the alert's button's border layer
+     */
+    public var buttonBorderWidth: CGFloat = 0.0
+    
+    /**
+     * UIColor of the alert's button's border
+     */
+    public var buttonBorderColor = UIColor.clear
+    
+    /**
+     * Opacity of the alert's button's shadows
+     */
+    public var buttonShadowOpacity: Float = 0.0
```

Comment:
Also, by encapsulating things like Shadow in a type, you can apply that to the button shadows too.